bin/deployment-cumulus-orca.sh

```diff
@@ -0,0 +1,272 @@
+#!/bin/bash
+set -ex
+
+export AWS_ACCESS_KEY_ID=$bamboo_AWS_ACCESS_KEY_ID
+export AWS_SECRET_ACCESS_KEY=$bamboo_AWS_SECRET_ACCESS_KEY
+export AWS_DEFAULT_REGION=$bamboo_AWS_DEFAULT_REGION
+
+#clone cumulus orca template for deploying cumulus and orca
+git clone https://git.earthdata.nasa.gov/scm/orca/cumulus-orca-deploy-template.git
+cd cumulus-orca-deploy-template
+git checkout $bamboo_ORCA_RELEASE_BRANCH
+echo "checked out to $bamboo_ORCA_RELEASE_BRANCH branch"
+
+
+#deploy data persistence tf module
+
+cd data-persistence-tf
+mv terraform.tfvars.example terraform.tfvars
+
+DATA_PERSISTENCE_KEY="$bamboo_PREFIX/data-persistence-tf/terraform.tfstate"
+
+# Ensure remote state is configured for the deployment
+echo "terraform {
+        backend \"s3\" {
+            bucket = \"$bamboo_PREFIX-tf-state\"
+            key    = \"$DATA_PERSISTENCE_KEY\"
+            region = \"$bamboo_AWS_DEFAULT_REGION\"
+            dynamodb_table = \"$bamboo_PREFIX-tf-locks\"
+    }
+}" > terraform.tf
+
+# Initialize deployment
+terraform init \
+  -input=false
+
+# Deploy data-persistence via terraform
+echo "Deploying Cumulus data-persistence module to $bamboo_DEPLOYMENT"
+terraform apply \
+  -auto-approve \
+  -lock=false \
+  -input=false \
+  -var-file="terraform.tfvars" \
+  -var "prefix=$bamboo_PREFIX" \
+  -var "aws_region=$bamboo_AWS_DEFAULT_REGION" \
+  -var "subnet_ids=[\"$bamboo_AWS_SUBNET_ID1\"]" \
+  -var "vpc_id=$bamboo_VPC_ID" \
+  -var "rds_user_access_secret_arn=$bamboo_RDS_USER_ACCESS_SECRET_ARN" \
+  -var "rds_security_group=$bamboo_RDS_SECURITY_GROUP"\
+  -var "permissions_boundary_arn=arn:aws:iam::$bamboo_AWS_ACCOUNT_ID:policy/$bamboo_ROLE_BOUNDARY"
```

Comment:
We could actually look this up instead of constructing it either in terraform with `data` or via the AWS Cli

Reply:
created https://bugs.earthdata.nasa.gov/browse/ORCA-464